### DISCUSSION TITLE
Unable to untap phinze/cask

### DISCUSSION THREAD

**Discussion by sitase** [Homebrew/discussions#702] [Casks]
I am trying to set up a new mac by using homebrew-file to sync the installed sw. Now, there are a number of formulae/casks/taps that have become stale over the years or just simply disappeared. One such is the tap phinze/cask whose function I guess has been absorbed by the main homebrew installation. So I try to untap it. That doesn't work since I have a number of installed casks. I am a bit stuck -- how do I get rid of phinze/cask while homebrew continues to handle these casks?

**Comment by SMillerDev** [comment]
If there is no definition of these casks homebrew can't manage them. If they are in `homebrew/cask` now, just change the name in your brewfile.
